identify/extensions.py

```diff
@@ -86,6 +88,7 @@
     'kml': {'text', 'kml', 'xml'},
     'kt': {'text', 'kotlin'},
     'less': {'text', 'less'},
+    'lhs': {'text', 'haskell'},
```

Comment:
is it common that linters / formatters would handle all three extensions the same? or should these produce a different tag

Reply:
It might be nice to have them separate. i just didn't know if that would be ... polluting the namespace too much.
They are quite a bit different.
Literate haskell lets you mix in compilable code with like a blog almost
c haskell does crazy C processing inline with the haskell itself.

I am happy to split them up!

Reply:
probably makes the most sense to split them if I had to guess